docs/mobile-apps/automated-testing/appium/appium-flutter-integration-driver.md

```diff
@@ -0,0 +1,678 @@
+{label: 'iOS', value: 'ios'},
+]}>
+<TabItem value="android">
+
+<!-- prettier-ignore -->
+```java
+MutableCapabilities capabilities = new MutableCapabilities();
+
+capabilities.setCapability("platformName", "android");
+// W3C Protocol is mandatory for Appium 2
+capabilities.setCapability("appium:platformVersion", "12");
+capabilities.setCapability("appium:deviceName", "Google Pixel 6");
+// Mandatory for Appium 2
+capabilities.setCapability("appium:automationName", "FlutterIntegration");
+capabilities.setCapability("appium:app", "storage:filename=sl_my_demo_flutter_app.apk");
+
+HashMap<String, Object> sauceOptions = new HashMap<String, Object>();
+// appiumVersion is mandatory to use Appium 2 on Sauce Labs
+sauceOptions.put("appiumVersion", "appium-20240701");
+capabilities.setCapability("sauce:options", sauceOptions);
+```
+
+</TabItem>
+<TabItem value="ios">
+
+<!-- prettier-ignore -->
+```java
+MutableCapabilities capabilities = new MutableCapabilities();
+
+capabilities.setCapability("platformName", "ios");
+// W3C Protocol is mandatory for Appium 2
+capabilities.setCapability("appium:platformVersion", "16");
+capabilities.setCapability("appium:deviceName", "iPhone 14");
+// Mandatory for Appium 2
+capabilities.setCapability("appium:automationName", "FlutterIntegration");
+capabilities.setCapability("appium:app", "storage:filename=sl_my_demo_flutter_app.ipa");
+
+HashMap<String, Object> sauceOptions = new HashMap<String, Object>();
+// appiumVersion is mandatory to use Appium 2 on Sauce Labs
+sauceOptions.put("appiumVersion", "appium-20240701");
+capabilities.setCapability("sauce:options", sauceOptions);
+```
+
+</TabItem>
+</Tabs>
+</TabItem>
+<TabItem value="js">
+
+<Tabs
+groupId="capability-js"
+defaultValue="android"
+values={[
+{label: 'Android', value: 'android'},
+{label: 'iOS', value: 'ios'},
+]}>
+<TabItem value="android">
+
+<!-- prettier-ignore -->
+```js
+const capabilities = {
+    platformName: 'android',
+    // W3C Protocol is mandatory for Appium 2
+    'appium:platformVersion': '12',
+    'appium:deviceName': 'Google Pixel 6',
+    // Mandatory for Appium 2
+    'appium:automationName': 'FlutterIntegration',
+    'appium:app': 'storage:filename=sl_my_demo_flutter_app.apk',
+    'sauce:options': {
+        // appiumVersion is mandatory to use Appium 2 on Sauce Labs
+        appiumVersion: 'appium-20240701'
+    }
+}
+```
+
+</TabItem>
+<TabItem value="ios">
+
+<!-- prettier-ignore -->
+```js
+const capabilities = {
+    platformName: 'ios',
+    // W3C Protocol is mandatory for Appium 2
+    'appium:platformVersion': '16',
+    'appium:deviceName': 'iPhone 14',
+    // Mandatory for Appium 2
+    'appium:automationName': 'FlutterIntegration',
+    'appium:app': 'storage:filename=sl_my_demo_flutter_app.ipa',
+    'sauce:options': {
+        // appiumVersion is mandatory to use Appium 2 on Sauce Labs
+        appiumVersion: 'appium-20240701'
+    }
+}
+```
+
+</TabItem>
+</Tabs>
+
+</TabItem>
+<TabItem value="python">
```

Comment:
java and wdio client bindings support only. can you remove everything that is not java/js binding?

Reply:
done